Firestore/core/src/firebase/firestore/immutable/sorted_map.h

```diff
@@ -264,6 +265,15 @@ class SortedMap : public SortedMapBase {
     UNREACHABLE();
   }
 
+  absl::optional<V> get(const K& key) const {
+    auto found = find(key);
+    if (found != end()) {
+      return found->second;
```

Comment:
Looks like you decided that returning a nested optional is a non-issue?

Reply:
Ah, just checking. I wasn't clear about what you're planning to do with `OptionalMaybeDocumentMap`.